.github/workflows/fc-asciidoctor-publish.yaml

```diff
@@ -0,0 +1,49 @@
+name: fc-asciidoctor-publish
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+    paths:
+      - utilities/fc-asciidoctor/version.json
+      - .github/workflows/fc-asciidoctor-publish.yaml
+jobs:
+  build:
+    env:
+      context: utilities/fc-asciidoctor
+      image_name: fc-asciidoctor
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+      - name: Get image tags
```

Comment:
this step can be released by the central action. see:
- https://github.com/redhat-cop/containers-quickstarts/blob/master/.github/workflows/build-s2i-python-kopf-publish.yaml#L18-L22

Reply:
Sorry, not making it easier for review.